devmapper/devmapper.go

```diff
@@ -501,7 +521,7 @@ func (d *Driver) CreateVolume(req Request) error {
 	}
 	if backupURL == "" {
 		// format the device
-		if _, err := util.Execute("mkfs", []string{"-t", "ext4", dev}); err != nil {
+		if err := d.createFilesystem(dev); err != nil {
```

Comment:
I see what you meant. Thanks for explaining it. I will simplify it based on your comments.